internal/handler/msg_createuser.go

```diff
@@ -115,11 +110,16 @@ func (h *Handler) MsgCreateUser(ctx context.Context, msg *wire.OpMsg) (*wire.OpM
 		return nil, lazyerrors.Error(err)
 	}
 
-	var credentials *types.Document
+	if mechanisms.Len() == 0 {
+		return nil, handlererrors.NewCommandErrorMsg(
+			handlererrors.ErrBadValue,
+			"mechanisms field must not be empty",
+		)
+	}
```

Comment:
We will want to check mechanisms are valid here. So we don't need to handle invalid mechanisms in backend

Reply:
Fixed :pray: